src/chocolatey/infrastructure.app/services/PowershellService.cs

```diff
@@ -403,8 +404,8 @@ public void prepare_powershell_environment(IPackage package, ChocolateyConfigura
 
             // we only want to pass the following args to packages that would apply. 
             // like choco install git -params '' should pass those params to git.install, 
-            // but not another package
-            if (!package_is_a_dependency_not_a_virtual(configuration, package.Id))
+            // but not another package unless the switch applyToDependentPackages is used
+            if (!package_is_a_dependency_not_a_virtual(configuration, package.Id) || configuration.ApplyToDependentPackages)

```

Comment:
It may not be a bad idea to split the arguments for applying install arguments away from package parameters.